usr/local/bin/um-setup

```diff
@@ -184,9 +184,11 @@ echo "Installing GCOM from: $gcom_src"
 cd $tmpdir/wc
 echo "Running rose stem to build GCOM..."
 
-rose stem --quiet --group=all --name=gcom_install --new -- --no-detach
+rose stem --quiet --group=all --name=gcom_install --new --no-gcontrol -- --no-detach
 if [ $? -ne 0 ]; then
-  echo "Failed to launch GCOM rose stem suite."
+  echo "Problem running GCOM rose stem suite."
+  echo "Check suite logs and output at /home/vagrant/cylc-run/gcom_install"
+  echo "or try http://localhost/rose-bush/jobs/$USER/gcom_install."
```

Comment:
This is a legacy URL. It should now be:
http://localhost/rose-bush/taskjobs/$USER?&suite=gcom_install
Ditto for URL below,

Also, any reason not to specify "vagrant" rather than $USER ?

Reply:
Paths updated, thanks for that. I noticed they'd changed in cylc review but not rose bush.

`$USER` because I don't trust the providers not to break things, especially after the [mess](https://github.com/hashicorp/vagrant/issues/9442) they made changing where boxes are stored; and because someone might want to do something clever or use the script as the basis for a generic UM install script one day. It doesn't refer to "vagrant" or `/home/vagrant` anywhere, only `$USER` and `$HOME`.